_rules/7d6734.md

```diff
@@ -0,0 +1,207 @@
+---
```

Comment:
the filename has to change to have rule name + unique id, something like `svg-image-name-7d6734`